packages/nuekit/src/layout/components.js

```diff
@@ -66,18 +65,22 @@ export function renderPage(page) {
   const h2 = title ? elem('h2', renderInline(title)) : ''
   const p = desc ? elem('p', renderInline(desc)) : ''
 
-  const body = !thumb ? time + elem('a', { href: url }, h2 + p) :
-
-    // figure
-    elem('a', { href: url }, elem('figure',
-      elem('img', { src: thumb, loading: 'lazy' }) + elem('figcaption', time + h2 + p))
-  )
+  const body = !thumb
+    ? time + elem('a', { href: url }, h2 + p)
+    : // figure
+      elem(
```

Comment:
I think we should stick to compact formatting in elem() calls consistently.

Reply:
Agreed. I'll have a look at the prettier config and see if I can enforce that 👍

Reply:
I spent some time looking at this and really there isn't a good way of doing it without building a prettier plugin. I've never done that and would be up for trying it if you're interested?